packages/sit-onyx/src/components/OnyxButton/OnyxButton.vue

```diff
@@ -147,5 +149,9 @@ const props = withDefaults(defineProps<ButtonProps>(), {
     font-weight: 600;
     line-height: 1.5rem;
   }
+
+  &__icon {
+    color: var(--onyx-button-text-color);
```

Comment:
I'd prefer to set the color directly on the "onyx-button" class instead of on the label only. Then we don't need to set it for the icon here because it will just inherit the font color